### ISSUE TITLE
Current ring.ogg is very disturbing

### ISSUE THREAD

**Issue by r4dh4l** [vector-im/element-android#2936]
Hi,

because I'm slightly oversenstive related to hearing the current [ring.ogg](https://github.com/vector-im/element-android/files/6057255/ring.ogg.zip) was always very disturbing for me. Recently I spoke with a friend about how ring.ogg sounds for others and the impression seems to be the same: The current ring.gg is charming like https://www.youtube.com/watch?v=BIKarAqOB9I .

I don't know if it is a bug on my F-Droid Element v1.1.0 but there the problem is even worse because I can not tune down the volume when ring.ogg is played during the dial process. However: May we discuss about another default?

Does it has to be something like an "alarm"? The current ring.ogg seems to be chosen to be something like https://freesound.org/people/kwahmah_02/sounds/250629/ or could it be something like https://soundbible.com/1795-Electrical-Sweep.html as well: Ascending and descending in loudness and much more gentle ?

**Comment by reg3xp** [comment]
Hi,
as i suggested in #2915 we can replace it with element-desktop's dial tone which is pleasant.

**Comment by sl5net** [comment]
yes ringing sound (if u call somebody) is terribly loud,useless, not adjustable, headache

Often misunderstood: it's not about the tone when someone calls, but about the tone when someone else is called (in other words, when you / I call someone else). Matrix Element ringing sound is terribly loud, and not adjustable, makes a headache https://youtu.be/OOulxYMrAPQ

**Comment by r4dh4l** [comment]
I already tried to replace `ring.ogg` with another audio file keeping the file name in Android but it didn't effect the ring tone. However: It would be great to have an option within Element to set a custom ring tone.

**Comment by sl5net** [comment]
helps me: https://play.google.com/store/apps/details?id=com.rannsoft.soundoff&hl=de&gl=US
